samples/client/petstore/java/jersey1/src/test/java/io/swagger/client/ApiClientTest.java

```diff
@@ -203,25 +246,12 @@ public void testParameterToPairsWhenValueIsEmptyStrings() throws Exception {
     }
 
     @Test
-    public void testParameterToPairsWhenValueIsNotCollection() throws Exception {
-        String name = "param-a";
-        Integer value = 1;
-
-        List<Pair> pairs = apiClient.parameterToPairs("csv", name, value);
-
-        assertEquals(1, pairs.size());
-        assertEquals(value, Integer.valueOf(pairs.get(0).getValue()));
-    }
-
-    @Test
     public void testParameterToPairsWhenValueIsCollection() throws Exception {
         Map<String, String> collectionFormatMap = new HashMap<String, String>();
         collectionFormatMap.put("csv", ",");
         collectionFormatMap.put("tsv", "\t");
         collectionFormatMap.put("ssv", " ");
-        collectionFormatMap.put("pipes", "\\|");
-        collectionFormatMap.put("", ","); // no format, must default to csv
-        collectionFormatMap.put("unknown", ","); // all other formats, must default to csv
```

Comment:
@michaelkourlas is it correct to say that the default is no longer csv (",") when unknown or no format is provided?